### ISSUE TITLE
problem with the /tmp/threed_front.pkl file

### ISSUE THREAD

**Issue by kkkkkkkkkkkkkkkin** [nv-tlabs/ATISS#25]
I've managed to setup everything and tried to run the preprocess_data.py script. As the dataset was about to finish loading, I was met with the error `FileNotFoundError: [Errno 2] No such file or directory: '/tmp/threed_front.pkl'`. 
This is how I ran the preprocess_data.py script:
```
python .\scripts\preprocess_data.py data\output\1 data\dataset\3D-FRONT data\dataset\3D-FUTURE-model data\dataset\3D-FUTURE-model\model_info.json \demo\floor_plan_texture_images --dataset_filtering threed_front_bedroom
Applying threed_front_bedroom filtering
Loading dataset  6812 /  6813
Traceback (most recent call last):
  File ".\scripts\preprocess_data.py", line 272, in <module>
    main(sys.argv[1:])
  File ".\scripts\preprocess_data.py", line 152, in main
    dataset = ThreedFront.from_dataset_directory(
  File "c:\users\mibig\desktop\atiss\scene_synthesis\datasets\threed_front.py", line 169, in from_dataset_directory
    scenes = parse_threed_front_scenes(
  File "c:\users\mibig\desktop\atiss\scene_synthesis\datasets\utils.py", line 129, in parse_threed_front_scenes
    pickle.dump(scenes, open("/tmp/threed_front.pkl", "wb"))
```

Is the threed_front.pkl file supposed to automatically generate, and did I miss any steps? 

The same error shows when I tried to pickle the 3D-FUTURE dataset.

```
python .\scripts\pickle_threed_future_dataset.py data\output\pickledData data\dataset\3D-FRONT data\dataset\3D-FUTURE-model data\dataset\3D-FUTURE-model\model_info.json --dataset_filtering threed_front_bedroom
Applying threed_front_bedroom filtering
Loading dataset  6812 /  6813
Traceback (most recent call last):
  File ".\scripts\pickle_threed_future_dataset.py", line 127, in <module>
    main(sys.argv[1:])
  File ".\scripts\pickle_threed_future_dataset.py", line 101, in main
    scenes_dataset = ThreedFront.from_dataset_directory(
  File "c:\users\mibig\desktop\atiss\scene_synthesis\datasets\threed_front.py", line 169, in from_dataset_directory
    scenes = parse_threed_front_scenes(
  File "c:\users\mibig\desktop\atiss\scene_synthesis\datasets\utils.py", line 129, in parse_threed_front_scenes
    pickle.dump(scenes, open("/tmp/threed_front.pkl", "wb"))
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/threed_front.pkl'
```

Sorry if this is a stupid mistake, I am a beginner in python / conda stuff.

**Comment by kkkkkkkkkkkkkkkin** [comment]
Previously, I was unable to access the /tmp/threed_front.pkl file from
`pickle.dump(scenes, open("/tmp/threed_front.pkl", "wb"))`
I changed the code to find the absolute path of my threed_front.pkl file and try to create it.
I also made tried to change the PATH_TO_SCENE variable to a new one, I also made a new environment variable according to its supposed path (to threed_front.pkl). At first it worked, but when I checked, the new env variable did not register and returned None.

Is the PATH_TO_SCENE's key variable changeable? I tried setting a Windows environment variable named PATH_TO_SCENE but it does not work. Is the variable defined with the python scripts?

Sorry for changing the code on my own.
